sudoku/easy/addRandom.js

```diff
@@ -0,0 +1,29 @@
+let div = document.querySelectorAll('.board>div');
+export function addRandom() {
```

Comment:
can you, please divide this function into several small functions with descriptive names? it will make the code much more understandable